cypress.json

```diff
@@ -0,0 +1,17 @@
+{
+  "baseUrl": "https://demo.realworld.io/#/",
+  "viewportWidth": 1200,
+  "chromeWebSecurity": false,
```

Comment:
I would remove these non default options:

- chromeWebSecurity
- modifyObstructiveCode
- watchForFileChanges

Did you have a good reason to add them?

Reply:
I would also remove the timeout options.

They make feedback slower when iterating but maybe they're required for github actions?